models/keyword_test.go

```diff
@@ -0,0 +1,123 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/gutakk/go-google-scraper/db"
+	"github.com/gutakk/go-google-scraper/tests"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/go-playground/assert.v1"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+func TestValidateFileTypeWithValidFileType(t *testing.T) {
+	result := ValidateFileType("text/csv")
+
+	assert.Equal(t, nil, result)
+}
+
+func TestValidateFileTypeWithInvalidFileType(t *testing.T) {
+	result := ValidateFileType("test")
+
+	assert.Equal(t, "File must be CSV format", result.Error())
+}
+
+func TestValidateCSVLengthWithOneRow(t *testing.T) {
+	result := ValidateCSVLength(1)
+
+	assert.Equal(t, nil, result)
+}
+
+func TestValidateCSVLengthWithOneThoudsandRow(t *testing.T) {
+	result := ValidateCSVLength(1000)
+
+	assert.Equal(t, nil, result)
+}
+
+func TestValidateCSVLengthWithZeroRow(t *testing.T) {
+	result := ValidateCSVLength(0)
+
+	assert.Equal(t, "CSV file must contain between 1 to 1000 keywords", result.Error())
+}
+
+func TestValidateCSVLengthWithMinusOneRow(t *testing.T) {
+	result := ValidateCSVLength(-1)
```

Comment:
How can this case even happen 🤔 ?

Reply:
Good catch. it is impossible. Fixed in b2b3d88f76e43e6cdf496f6033eac0fce9a58c05